src/components/tracked-controls-webxr.js

```diff
@@ -62,17 +62,25 @@ module.exports.Component = registerComponent('tracked-controls-webxr', {
   },
 
   emitButtonDownEvent: function (evt) {
-    if (!this.controller || evt.inputSource.handedness !== this.data.hand) { return; }
+    if (!this.controller) { return; }
     if (this.controller.gamepad) { return; }
+    if (evt.inputSource.handedness !== 'none' &&
```

Comment:
This shoud not be necessary or handled somewhere else. Can you explain why this is needed?